cmake/Dependencies.common.cmake

```diff
@@ -57,7 +57,7 @@ endif()
 ##################################################################
 if (BUILD_JPEG_TURBO)
   find_package(JPEG 62 REQUIRED) # 1.5.3 version
-  include_directories(SYSTEM ${JPEG_INCLUDE_DIR})
```

Comment:
Could you run the CI with BUILD_AARCH64_LINUX TEST_AARCH64_LINUX and BUILD_AARCH64_QNX to see if there are no problem with this?

Reply:
sure